src/include/mallocMC/creationPolicies/Scatter.hpp

```diff
@@ -767,10 +767,13 @@ namespace mallocMC
             {
                 if(bytes == 0)
                     return 0;
-                // take care of padding
-                // bytes = (bytes + dataAlignment - 1) & ~(dataAlignment-1); //
-                // in alignment-policy
-                if(bytes < pagesize)
+                /* Take care of padding
+                 * bytes = (bytes + dataAlignment - 1) & ~(dataAlignment-1);
+                 * in alignment-policy.
```

Comment:
Are these 3 lines relevant to this part? I know this PR doesn't change it, but probably they belong somewhere else?

Reply:
I kept the equation separated from the normal text that it is better readable.